client/src/components/ResponseHandler.tsx

```diff
@@ -35,12 +35,13 @@ function ResponseHandler() {
     if (response.ok) {
       const data = await response.json();
       let message = data.response?.message ?? "";
-      let messages = data.response?.messages ?? [];
-      let options = data.response?.options ?? [];
-      let system = data.response?.system ? "\n> " + data.response.system : null;
+      let options = (data.response?.options ?? []).map((option: any[]) => option.join(': '));
 
       console.log(data);
-      setConversation((prev) => [...prev, "\nUser: "+userInput+"\n", message, ...options, ...messages, system]);
+      setConversation((prev) => [...prev, "\nUser: "+userInput+"\n", message, ...options]);
+      //if (!data.response?.options){
+      //  handleUserInput("");
+      //}
```

Comment:
Remove dead code!

Reply:
Yep, this is here as a TODO for the next PR but thats bad practice, should've just left a comment

Reply:
Makes sense. Didn't have the contextual knowledge to realize that. That's more ok :)